### ISSUE TITLE
Change image-span on index page to 3 images in a row with captions underneath

### ISSUE THREAD

**Issue by MillieMD** [MillieMD/Edu-Code#8]
Text in images is not accessible to screen readers, convert the text to plain HTML

**Comment by MillieMD** [comment]
Additionally, the text in the image uses the wrong name so it needs changing anyway

**Comment by MillieMD** [comment]
Reverted by accident
